tests/karma/BookNavigator/volumes/volumes-provider.test.js

```diff
@@ -56,6 +59,22 @@ describe('Volumes Provider', () => {
     expect(provider.component).to.exist;
   });
 
+  describe('sorting cycles - render headerIcon actionButton', async () => {
+    const onSortClick = sinon.fake();
+    const baseHost = "https://archive.org";
+    const provider = new volumesProvider(onSortClick, baseHost, brOptions);
+
+    provider.sortVolumes("initial");
+    expect(provider.headerIcon.getIcon.getHTML()).includes("sort-by neutral-icon");
+
+    provider.sortVolumes("asc");
+    expect(provider.headerIcon.getIcon.getHTML()).includes("sort-by asc-icon");
+
+    provider.sortVolumes("desc");
+    expect(provider.headerIcon.getIcon.getHTML()).includes("sort-by desc-icon");
```

Comment:
Oh, and question: What is `getIcon` here? I though `headerIcon` was a ``html`<button ... />`;``, so where is the `getIcon` method coming from?

Reply:
`getIcon` was a method from the `TemplateResult` since `get headerIcon` returns a `TemplateResult`